blogpost/Blogpost-draft.md

```diff
@@ -1,39 +1,268 @@
-# Jenkins 2.0 Pipelines
-## Subheading with even more buzzwords
+# JENKINS 2.0 (PART 2) - PIPELINES IN ACTION 
+## How to manage your delivery pipeline as code
 
-2-3 sentence abstract
-* Local CD environment vagrant, ansible
-* Pre-configured Jenkins Pipeline
-* Starting point to play with pipelines in production-like scenario
+----
 
-### Intro / Local environment
-* Short explanation of local sandbox infrastructure
-* Maybe a small/simple diagram showing VMs + Networking + App
-* Links to repos
+_Read our thoughts and experiences we made with Jenkins Pipelines in this blog post, or directly checkout our [github repository](https://github.com/comsysto/jenkins2-pipeline-lab) and see it in action._
 
-### Continuous Delivery scenario concept
-* Short explanation of the CD scenario we build
-* Maybe visualisation of stages + steps
+---- 
 
-### Jenkins pipeline + Jenkins file explained
 
+### Jenkins Pipelines and Jenkinsfiles
+<a name=jenkinsPipelines/> 
+In the last blogpost about Jenkins 2.0 **[LINK]**, which focused on the configuration and administration of Jenkins 2.0, 
+we gave a quick overview of the new Pipline plugin and its features. 
+Now we want to have a closer look at the capabilities of the Pipeline plugin and work out the major innovations and differences that are introduced by the plugin. 
 
+#### Freestyle Jobs vs. Pipelines
+<a name=freestyleVsPipelines/> 
+
+The conventional way of building applications with Jenkins is to define a series of sequential executable steps, each configured as a freestyle job, and then trigger one
+after the other until the whole build process is completed. The chaining of freestyle jobs is usually done via the Jenkins UI, which makes changes and reconfigurations of 
+the job chain rather inconvenient. But fear not, the Jenkins Pipeline plugin is here for the rescue! 
```

Comment:
> The chaining of freestyle jobs is usually done via the Jenkins UI, which makes changes and reconfigurations of 
+the job chain rather inconvenient. But fear not, the Jenkins Pipeline plugin is here for the rescue! 
I'd reformulate this paragraph a little, like:

> The chaining of freestyle jobs is usually done via the Jenkins UI, a cumbersome process that makes changes and reconfiguration of the job chain rather inconvenient. To alleviate this issue, Jenkins 2.0 introduces the Pipeline plugin.